common/utils/src/main/resources/error/error-classes.json

```diff
@@ -6958,6 +6958,21 @@
       "<message>"
     ]
   },
+  "_LEGACY_ERROR_TEMP_3101" : {
+    "message" : [
+      "The input is not a correct window column: <windowTime>"
+    ]
+  },
+  "_LEGACY_ERROR_TEMP_3102" : {
```

Comment:
There are already 5 error message templates:  _LEGACY_ERROR_TEMP_3064, _LEGACY_ERROR_TEMP_3066, _LEGACY_ERROR_TEMP_3071, _LEGACY_ERROR_TEMP_3072, _LEGACY_ERROR_TEMP_3080, which are as  

```
"message" : [
      "<msg>"
    ]
```
...

Reply:
We cannot merge them because those are different kind of errors. Later we should improve them.

Reply:
OK ~